en/config_fw/weight_and_altitude_tuning.md

```diff
@@ -0,0 +1,127 @@
+
+E.g. by $\hat{\dot{h}}_{max}$ we specify the maximum climb rate the vehicle can achieve at [WEIGHT_BASE](../advanced_config/parameter_reference.md#WEIGHT_BASE) at sea level in standard atmospheric conditions.
+
+## Compensating for Weight
+
+[WEIGHT_BASE](../advanced_config/parameter_reference.md#WEIGHT_BASE) specifies the base weight of the vehicle at which the [TECS tuning](position_tuning_guide_fixedwing.md#tecs-tuning-altitude-and-airspeed) was performed.
+The weight of the vehicle in the tuning configuration should be measured with a scale and the parameter should be set accordingly.
+
+[WEIGHT_GROSS](../advanced_config/parameter_reference.md#WEIGHT_BASE) specifies the actual weight of the vehicle at any given time.
+This parameter can be set to reflect e.g. a payload attached to the vehicle which was not present during the tuning phase.
+
+When both [WEIGHT_BASE](../advanced_config/parameter_reference.md#WEIGHT_BASE) and [WEIGHT_GROSS](../advanced_config/parameter_reference.md#WEIGHT_GROSS) are set to a number larger than 0 PX4 will use this information to:
+
+### Scale the Maximum Climb Rate
+
+The maximum climb rate ([FW_T_CLMB_MAX](../advanced_config/parameter_reference.md#FW_T_CLMB_MAX)) must be scaled as a function of the weight ratio.
+
+From the steady state equations of motions of an airplane we find that the maximum climb rate can be written as
+
+$$ \dot{h}_{max} = { V * ( Thrust - Drag ) \over{m*g}}  $$
+
+where V is the true airspeed and m is the vehicle mass.
+Therefore, we can write the maximum climb rate as
+
+$$ \dot{h}_{max} = \hat{\dot{h}}_{max} * {m_{base} \over{m_{gross}}} $$
+
+### Scale the Minimum Sink Rate
+
+The minimum sink rate ([FW_T_SINK_MIN](../advanced_config/parameter_reference.md#FW_T_SINK_MIN)) must be scaled as a function of weight ratio
+
+The minimum sink rate can be written as:
+
+$$ \dot{h}_{min} = \sqrt{2mg\over{\rho F}} f(c_A, c_W) $$
+
+where $\rho$ is the air density, F is the wing area and f(c_A, c_W) is a function of the polars.
+Therefore, we can compute the minimum sink rate as:
+
+$$ \dot{h}_{min} = \hat{\dot{h}}_{min}  \sqrt{m_{gross}\over{m_{base}}} $$
+
+### Adjust Minimum, Stall, and Trim Airspeed Limits According to the Weight Ratio
+
+Next adjust airspeed limits such as minimum airspeed ([FW_AIRSPD_MIN](../advanced_config/parameter_reference.md#FW_AIRSPD_MIN)), the stall airspeed ([FW_AIRSPD_STALL](../advanced_config/parameter_reference.md#FW_AIRSPD_STALL)) and trim airspeed ([FW_AIRSPD_TRIM](../advanced_config/parameter_reference.md#FW_AIRSPD_TRIM)) according to the weight ratio.
+
+In steady state flight we can demand that lift should equal weight of the vehicle:
+
+$$ Lift = mg = {1\over{2}} \rho c_A F V^2 $$
+
+rearranging this equation for airspeed gives:
+
+$$ V = \sqrt{{2mg}\over{\rho c_A F}} $$
```

Comment:
This breaks rendering. According to ChatGPT, you probably mean the following expression (which seems to work, and in any case does not break rendering).

```suggestion
$$ V = \\sqrt{\\frac{2mg}{\\rho c_A F}} $$
```